### DISCUSSION TITLE
How do I use SteamTinkerLaunch as the Wine version?

### DISCUSSION THREAD

**Discussion by sylvesterroos** [Heroic-Games-Launcher/HeroicGamesLauncher#1784] [Q&A]
If I wanted to use this tool with Proton games on Steam I would set SteamTinkerLaunch as the compatibility tool. In Heroic I do not have this option (I've not seen it anyway). Is there a way to get this to work? I need it to mod my CP2077 with Vortex

**Comment by DumbledoreCalrissian** [comment]
I'd like to know this too!

**Reply by jtyoboy** [reply]
same here
